src/log/message.ts

```diff
@@ -26,6 +26,9 @@ export function invalidSpec(spec: GenericSpec<any, any>) {
 
 // FIT
 export const FIT_NON_SINGLE = 'Autosize "fit" only works for single views and layered views.';
+export const CONTAINER_SIZE_NON_SINGLE = 'Width/height "container" only works for single views and layered views.';
```

Comment:
If we know if it is width or height that's problematic, we should make that clear. (We can make this log a function that takes `sizeType: 'width' | 'height'` as an input.)

Reply:
Actually this isn't being used right now. Is there a function to determine if the view is single or layered?

Reply:
I tried `isLayerSpec(spec) || isUnitSpec(spec)` like in the `FIT_NON_SINGLE` warning, but it didn't work for column/row encodings.

(btw, this should be a separate issue: the `FIT_NON_SINGLE` warning doesn't trigger for column/row encodings, and the chart won't render. To reproduce, you can add `autosize: "fit"` to https://vega.github.io/editor/#/examples/vega-lite/trellis_stacked_bar)